test/e2e/atlas/setup_force_test.go

```diff
@@ -56,13 +56,40 @@ func TestSetup(t *testing.T) {
 			"--skipSampleData",
 			"--projectId", g.projectID,
 			"--tag", tagKey+"="+tagValue,
+			"--accessListIp", randomAccessListIP,
 			"--force")
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
 		req.NoError(err, string(resp))
 		assert.Contains(t, string(resp), "Cluster created.", string(resp))
 	})
 
+	t.Run("Check accessListIp was correctly added", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			accessListEntity,
+			"ls",
+			"--projectId",
+			g.projectID,
+			"-o=json")
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		req.NoError(err)
+
+		var entries *atlasv2.PaginatedNetworkAccess
+		err = json.Unmarshal(resp, &entries)
+		req.NoError(err)
+
+		found := false
+		for i := range entries.Results {
+			if entries.Results[i].GetIpAddress() == randomAccessListIP {
+				found = true
+				break
+			}
+		}
+
+		req.True(found)
```

Comment:
```suggestion
		req.Contains(entries.Results[i].GetIpAddress(), randomAccessListIP)
```
https://pkg.go.dev/github.com/stretchr/testify@v1.8.4/assert#Contains